tests/integration-service/README.md

```diff
@@ -0,0 +1,154 @@
+
+- Verifying that all integration pipeline runs finish successfully.
+- Checking that a snapshot gets successfully marked as 'passed' after specific events.
+
+
+### Environment and Deployment Scenarios:
+
+- Ensuring that environments are created successfully.
+- Verifying the existence of deploymentTargetClass.
+
+
+## Negative Test Cases.
+
+### Failure Scenarios:
+
+- Creating an IntegrationTestScenario that is expected to fail.
+- Asserting that a snapshot is marked as failed.
+- Validating that no Release CRs and no SnapshotEnvironmentBinding are created in certain scenarios.
+- Checking that the global candidate does not get updated unexpectedly.
+
+
+### Non-existent Scenarios:
+- Verifying that entities like deploymentTargetClass and GitOpsCR don't exist under certain conditions.
+- Asserting that no GitOpsCR is created for a non-existing deploymentTargetClass.
+- Checking that a snapshot doesn't get marked as 'passed' under specific conditions.
+  
+
+# Running e2e tests.
+
+## Prerequisites:
+
+- Go workspace using Go 1.19 required
+- OpenShift command line tool (oc). Ensure command line version matches OpenShift version.
+- [yq](https://mikefarah.gitbook.io/yq/)
+- [jq](https://jqlang.github.io/jq/)
+- git
+- [helm](https://helm.sh/)
+- [Github organisation](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/creating-a-new-organization-from-scratch) with [access token](https://docs.github.com/en/enterprise-server@3.6/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens) that has permissions: write, delete
+- Fork of [infra-deployments](https://github.com/redhat-appstudio/infra-deployments) in github organisation
+- [Quay.io organisation](https://access.redhat.com/documentation/en-us/red_hat_quay/3.3/html/use_red_hat_quay/proc-use-quay-create-user-org), existing organisation cannot be used. Create one new organisation for testing purposes only. 
+- Repository in new quay.io organisation called test-images. The repository name must follow this format: OrganisationName/test-images
+- Access token for quay.io organisation. Create in quay.io at 
+```bash
+organisation > repository name > applications > create new application
+```
+- Check that ~/.docker/config.json file is in this format: 
+
+```bash
+{
+  "auths": {
+    "quay.io": {
+      "auth": "------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------==",
+      "email": ""
+    }
+  }
+}
+```
+
+## Setting up the cluster
+
+- ClusterBot - set up as normal until preview mode step. Do not set up preview mode, go to step 1 below. 
+- QuickCluster - setup up to and including setup-nfs-quickcluster.sh then go to step 1 below.
+- [Instructions for setting up clusters](https://docs.google.com/document/d/1EfJE563dUimxxp1xTLeqjq1s0PUyO3G0UCtaR8rTi24)
```

Comment:
I don't know if we should be adding links to internal google docs files on public GitHub repos.  In fact, I'm not even sure we should mention anything about how to set up ClusterBot or QuickCluster since those are internal resources.  @dirgim what are your thoughts?